custom_dc/data_cloud.md

```diff
@@ -120,7 +121,7 @@ gcr.io/datcom-ci/datacommons-website-compose:stable
 
 #### Run with a locally built repo
 
-If you have made local changes and have a [locally built repo](build_repo.md), from the `website` root of the repository, run the following:
+If you have made local changes and have a [locally built repo](custom_dc/build_repo.html), from the `website` root of the repository, run the following:
 
```

Comment:
This results in a double custom_dc/custom_dc url for me. Because this file and build_repo.md are in the same folder, I don't think you want the custom_dc in front.
```suggestion
If you have made local changes and have a [locally built repo](build_repo.html), from the `website` root of the repository, run the following:
```

Reply:
No, what I need here is /custom_dc (I missed the slash). Let me fix it.